iac-common/src/main/java/org/sonar/iac/common/checks/policy/Policy.java

```diff
@@ -61,7 +61,7 @@ public static class Statement {
     private final Tree notResource;
     private final Tree condition;
 
-    private Statement(Tree statement) {
+    public Statement(Tree statement) {
```

Comment:
```suggestion
    private Statement(Tree statement) {
```
I think, we can revert this.